.github/workflows/image.yaml

```diff
@@ -28,11 +28,11 @@ jobs:
             username: ${{ secrets.BOT_NAME }}
             password: ${{ secrets.BOT_TOKEN }}
 
-        - name: make container
-          run: make docker-build IMG=${OPERATOR_IMAGE}:latest
+        - name: Build Operator Image
+          run: make operator-build OPERATOR_IMG=${OPERATOR_IMAGE}:latest
```

Comment:
Offtopic: we should fix `latest` to a more version based on git-sha and datetime
filed: https://github.com/sustainable-computing-io/kepler-operator/issues/109